### PR TITLE
Added function calculateFileNameForGitHub

### PR DESCRIPTION
**Pull request by shekhar10feb**
## Description
Added calculateFileNameForGitHub function with date format as suggested by Proful in the issue #880 

## Attach Screenshot

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

**Comment by proful** [comment]
@PuruVJ and @Mira-Alf please review

@shekhar10feb please clean up the commit history.

https://github.com/twindle-co/twindle/blob/main/docs/articles/solving-branch-ahead-of-upstream.md

File: twindle-cli/src/index.js [hunk 1]
```diff
@@ -120,6 +129,15 @@ function calculateFileName(cliObject, data) {
   return intelligentOutputFileName;
 }
 
+function calculateFileNameForGitHub(cliObject, data) {
+   let userName = data[0].common.user.username;
```
Comment (PuruVJ) [inline twindle-cli/src/index.js:133]: In the Twitter part when we're doing the intelligent filename generation, a statement like this breaks down badly.

I don't know if this is the case for the github API, but I would request @Mira-Alf who knows of that intelligentFileName function as well the github API intricasies to look at it

**Review by PuruVJ — APPROVED** [review]
This can be approved. However, take note of my comment and someone please fix it up later on.

**Comment by PuruVJ** [comment]
@shekhar10feb I don't think you should apply the commit cleanup code in the article for this PR, as your changes might get deleted. Just take care for future commits

**Pull request merged**